src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    color: inherit;
+    transition: all $transition-time;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

Comment:
Although changing the card title text color is implemented correctly, it's recommended to use a variable for the color value. Define '$hover-card-title-color: #34568b;' in your _variables.scss file and use the variable here.